generators/server/templates/src/main/java/package/service/UserService.java.ejs

```diff
@@ -953,7 +953,7 @@ public class UserService {
      * @return the user from the authentication.
      */
         <%_ if (databaseType === 'sql') { _%>
-    @Transactional(readOnly = true)
+    @Transactional(readOnly = false)
```

Comment:
This is only necessary when `databaseType !== 'no'`. 
Also `readOnly` is `false` by default, so we should just have:
```suggestion
    @Transactional
```